pset2/README.md

```diff
@@ -3,7 +3,7 @@
 ## Instructors
 
 ### Taq Karim
-<img src='images/taq.png' width='150'>
+<img src='images/belle_and_me.jpg' width='150'>
```

Comment:
hey taq! why didn't you use the markdown syntax for this...?

Reply:
hey taq! it's becayse the image is super large and I wanted to assert a max width